### DISCUSSION TITLE
Upgrading Chia while plotting - Windows

### DISCUSSION THREAD

**Discussion by Ocidentally** [swar/Swar-Chia-Plot-Manager#189] [Q&A]
Since I’m in the middle of plotting 12 parallel plots using Swar, is there any way to upgrade to Chia 1.1.5 without breaking those plots?

**Comment by raid5** [comment]
I'm running Ubuntu Server and upgraded to 1.1.5 this morning without any issues.

**Reply by Ocidentally** [reply]
Unfortunately, I'm not on Linux (though I realize there are advantages - like this!)

**Reply by raid5** [reply]
Since you're using Swar for managing the plot creation, I believe you should still be fine on Windows to upgrade. The plots will continue on their own.

**Reply by Ocidentally** [reply]
Fair warning: I just tried upgrading and it broke all my plots. Have to start from zero. Not completely unexpected but wish Windows installations could make upgrading work better.

**Reply by GForceWeb** [reply]
@Ocidentally - This is likely due to the fact the Windows installer installs the new version in a version specific folder and then removes the existing version folder which the plotter would have been actively trying to use. In theory, it would be possible to keep plotting the existing plots on the old version if the folder wasn't deleted by the installer but it seems like best practice to play it safe and just ensure all plots are stopped for version upgrades.
